test/OpenTelemetry.Instrumentation.AspNetCore.Tests/RouteTests/README.net6.0.md

```diff
@@ -0,0 +1,550 @@
+# Test results for ASP.NET Core 6
+
+| Span http.route | Metric http.route | App | Test Name |
+| - | - | - | - |
+| :broken_heart: | :broken_heart: | ConventionalRouting | [Root path](#conventionalrouting-root-path) |
+| :broken_heart: | :broken_heart: | ConventionalRouting | [Non-default action with route parameter and query string](#conventionalrouting-non-default-action-with-route-parameter-and-query-string) |
+| :broken_heart: | :broken_heart: | ConventionalRouting | [Non-default action with query string](#conventionalrouting-non-default-action-with-query-string) |
+| :broken_heart: | :broken_heart: | ConventionalRouting | [Not Found (404)](#conventionalrouting-not-found-404) |
+| :broken_heart: | :green_heart: | ConventionalRouting | [Route template with parameter constraint](#conventionalrouting-route-template-with-parameter-constraint) |
+| :broken_heart: | :green_heart: | ConventionalRouting | [Path that does not match parameter constraint](#conventionalrouting-path-that-does-not-match-parameter-constraint) |
+| :broken_heart: | :broken_heart: | ConventionalRouting | [Area using area:exists, default controller/action](#conventionalrouting-area-using-areaexists-default-controlleraction) |
+| :broken_heart: | :broken_heart: | ConventionalRouting | [Area using area:exists, non-default action](#conventionalrouting-area-using-areaexists-non-default-action) |
+| :broken_heart: | :broken_heart: | ConventionalRouting | [Area w/o area:exists, default controller/action](#conventionalrouting-area-wo-areaexists-default-controlleraction) |
+| :green_heart: | :green_heart: | AttributeRouting | [Default action](#attributerouting-default-action) |
+| :green_heart: | :green_heart: | AttributeRouting | [Action without parameter](#attributerouting-action-without-parameter) |
+| :green_heart: | :green_heart: | AttributeRouting | [Action with parameter](#attributerouting-action-with-parameter) |
+| :green_heart: | :green_heart: | AttributeRouting | [Action with parameter before action name in template](#attributerouting-action-with-parameter-before-action-name-in-template) |
+| :green_heart: | :green_heart: | AttributeRouting | [Action invoked resulting in 400 Bad Request](#attributerouting-action-invoked-resulting-in-400-bad-request) |
+| :broken_heart: | :broken_heart: | RazorPages | [Root path](#razorpages-root-path) |
+| :broken_heart: | :broken_heart: | RazorPages | [Index page](#razorpages-index-page) |
+| :broken_heart: | :broken_heart: | RazorPages | [Throws exception](#razorpages-throws-exception) |
+| :green_heart: | :green_heart: | RazorPages | [Static content](#razorpages-static-content) |
```

Comment:
@alanwest How come `net6.0` does not have minimal API?

Reply:
Maybe I'm mistaken, but minimal API was introduced in .NET 7, right? A the very least the `MapGroup` API is not available in `net6.0`.

https://github.com/open-telemetry/opentelemetry-dotnet/pull/4950/files#diff-abac45368f3834763ae62630ef06c00b2ab5977a01dd4a240c3afc190a3bc28cR137-R139

Reply:
it is available in `.net6.0`. we have tests using that e.g. https://github.com/open-telemetry/opentelemetry-dotnet/blob/4a3c8d36b3d1dda97d0f3e0721e502619ddfe83a/test/OpenTelemetry.Instrumentation.AspNetCore.Tests/InProcServerTests.cs#L42

Reply:
cool, thanks! yea, I just lost track of when it was it was introduced.